### PR TITLE
Change the way of config offline mirror package

### PR DESCRIPTION
**Pull request by Win-Man**
### What is changed, added or deleted? (Required)

Change the way of config offline mirror package.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

**Comment by Win-Man** [comment]
@lonng PTAL

File: upgrade-tidb-using-tiup-offline.md [hunk 1]
```diff
@@ -11,13 +11,13 @@ aliases: ['/docs-cn/dev/upgrade-tidb-using-tiup-offline/']
 
 如果用户希望升级更新本地的 TiUP 离线镜像，可以参考[使用 TiUP 离线部署 TiDB 集群](/production-offline-deployment-using-tiup.md)的步骤 1 与步骤 2 下载部署新版本的 TiUP 离线镜像。在执行 `local_install.sh` 后，TiUP 会完成覆盖升级。
 
-随后，需要按照 `local_install.sh` 执行的结果，重新声明全局环境变量，并将 TIUP_MIRRORS 指向执行 `local_install.sh` 命令时输出的离线镜像包的位置 `/path/to/mirror`。
+随后，需要按照 `local_install.sh` 执行的结果，重新声明全局环境变量，并通过 `tiup mirror set` 命令设置 `local_install.sh` 命令时输出的离线镜像包的位置 `/path/to/mirror`。
```
Comment (lonng) [inline upgrade-tidb-using-tiup-offline.md:14]: 现在 `local_install.sh` 脚本会自动调用 `tiup mirror set`，所以我们可以直接删掉这几句。

Comment (Win-Man) [reply upgrade-tidb-using-tiup-offline.md]: 好的，我把这几行删掉了

**Pull request merged**